build.gradle

```diff
@@ -1,5 +1,6 @@
 plugins {
     id 'java'
```

Comment:
maybe we should switch this line to double quotes, to be consistent with the rest of the file?